App/Containers/Cafe.tsx

```diff
@@ -56,25 +64,50 @@ const ServiceTitle: TextStyle = {
   textAlign: 'left'
 }
 
-const ButtonContainer: ViewStyle = {
+const ExpirationDate: TextStyle = {
```

Comment:
Do me a favor and update this TextStyle to use the text styles defined in `styles/textStyle`. That will go a long way in making the screens you create look consistent with the rest of the app.

Reply:
Totally, on it.